docs/quickstart/cross-chain-transfers.md

```diff
@@ -0,0 +1,167 @@
+# Cross Chain Transfers
+
+## Introduction
+
+This article shows how to transfer AVAX tokens programmatically between any two chains of the Primary Network.
+
+## Requirements
+
+You've completed [Run an Avalanche Node](../nodes/build/run-avalanche-node-manually.md) and are familiar with [Avalanche's architecture](../overview/getting-started/avalanche-platform.md). You are familiar with and have cloned the [AvalancheJS](https://github.com/ava-labs/AvalancheJS) repo.
+
+In order to send AVAX, you need to have some AVAX! You can get real AVAX by buying it on an exchange, or you can get testnet AVAX from the [AVAX Test Faucet](https://faucet.avax.network), which is a free and easy way to get to play around with Avalanche.
+
+
+## Transferring AVAX using AvalancheJS
+
+The easiest way to transfer AVAX between chains is to use [AvalancheJS](https://github.com/ava-labs/AvalancheJS) which is a programmatic way to access and move AVAX.
+
+### Step 1 - Open AvalancheJS
+
+![Image for post](/img/examples.png)
+
+Select the **examples** folder to view the source code for AJS examples. To send AVAX between the Chains see the following:
+
+| Source Chain | Title | Path |
+| :--- | :--- | :--- |
+| ***X-Chain*** | [**AVM : Export Avax to C-Chain**](https://github.com/ava-labs/avalanchejs/blob/master/examples/avm/buildExportTx-cchain-avax.ts) | _X Chain >> C Chain_ |
+| ***X-Chain*** | [**EVM : Import Avax from X-Chain**](https://github.com/ava-labs/avalanchejs/blob/master/examples/evm/buildImportTx-xchain.ts) | _X Chain >> C Chain_ |
```

Comment:
Suggestion: let's avoid use the term AVM/EVM/PlatformVM, instead use X-Chain/C-Chain/P-Chain here.